src/Razor/src/Microsoft.VisualStudio.Editor.Razor/Telemetry/TelemetryReporter.cs

```diff
@@ -95,7 +95,9 @@ private static void AddToProperties(IDictionary<string, object?> properties, Pro
         }
 
         static bool IsComplexValue(object? o)
-            => o is Type type && Type.GetTypeCode(type) == TypeCode.Object;
+        {
+            return o?.GetType() is Type type && Type.GetTypeCode(type) == TypeCode.Object;
+        }
```

Comment:
Note that there's a rule in our .editorconfig to prefer braces for local methods, so I added them back.

Reply:
What is the advantage of using `GetTypeCode` here vs say `type == typeof(object)`?

Reply:
From the documentation for TypeCode.Object: "A general type representing any reference or value type not explicitly represented by another TypeCode."

`type == typeof(object)` wouldn't match arbitrary reference types.